myhoard/basebackup_operation.py

```diff
@@ -310,6 +313,9 @@ def _process_output_line(self, line):
             and not self._process_output_line_file_finished(line)
             and not self._process_output_line_lsn_info(line)
         ):
+            if "log block numbers mismatch" in line:
+                self.has_block_mismatch = True
```

Comment:
nit: probably does not make much sense to do it here, we could save some CPU pennies if we move it to the next block where we know it's an error.